packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEditForm.tsx

```diff
@@ -1 +1,151 @@
-export const RequestLineEditForm = () => {};
+import React from 'react';
+import {
+  Grid,
+  InputWithLabelRow,
+  NumericTextInput,
+  TextArea,
+  Typography,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { DraftResponseLine } from './hooks';
+
+interface ResponseLineEditFormProps {
+  disabled: boolean;
+  update: (patch: Partial<DraftResponseLine>) => void;
+  draftLine: DraftResponseLine;
+}
+
+const InfoRow = ({ label, value }: { label: string; value: string }) => {
+  return (
+    <Grid spacing={4} container direction="row">
+      <Grid xs={2} item>
+        <Typography variant="body1" fontWeight={700}>
+          {label}
+        </Typography>
+      </Grid>
+      <Grid xs={2} item>
+        <Typography
+          variant="body1"
+          sx={{
+            width: '220px',
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+            whiteSpace: 'nowrap',
+          }}
+        >
+          {value}
+        </Typography>
+      </Grid>
+    </Grid>
+  );
+};
+
+interface ResponseLineEditFormLayoutProps {
+  Left: React.ReactElement;
+  Middle: React.ReactElement;
+  Right: React.ReactElement;
+}
+
+export const ResponseLineEditFormLayout = ({
+  Left,
+  Middle,
+  Right,
+}: ResponseLineEditFormLayoutProps) => {
+  return (
+    <Grid
+      container
+      spacing={2}
+      direction="row"
+      justifyContent="space-between"
+      bgcolor="background.toolbar"
+      padding={2}
```

Comment:
```suggestion
      padding={4}
```
is more like the zeplin version - looks better with a bit more padding to me